src/libs/intents/localMethods.spec.ts

```diff
@@ -59,23 +62,32 @@ test('fetchSessionCode should return only a session code', async () => {
   fetchSessionCode.mockResolvedValue({
     session_code: 'test_session_code'
   })
-  const client = { getStackClient: () => ({ fetchSessionCode }) }
+  const client = {
+    getStackClient: (): { fetchSessionCode: jest.Mock } => ({
+      fetchSessionCode
+    })
+  }
 
-  const result = await localMethods(client).fetchSessionCode()
+  const result = await localMethods(client as CozyClient).fetchSessionCode()
 
   expect(fetchSessionCode).toHaveBeenCalledTimes(1)
   expect(result).toEqual('test_session_code')
 })
 
 test('fetchSessionCode should throw if no session code is returned', async () => {
   const fetchSessionCode = jest.fn()
-  const client = { getStackClient: () => ({ fetchSessionCode }) }
-
+  const client = {
+    getStackClient: (): { fetchSessionCode: jest.Mock } => ({
+      fetchSessionCode
+    })
+  }
   fetchSessionCode.mockResolvedValue({
     twoFactorToken: 'token'
   })
 
-  await expect(localMethods(client).fetchSessionCode()).rejects.toThrowError(
+  await expect(
+    localMethods(client as CozyClient).fetchSessionCode()
```

Comment:
nit: I would prefer to put the `as CozyClient` on the declaration (`const client = ...`)

Reply:
Updated